### DISCUSSION TITLE
Question about `register(name)`

### DISCUSSION THREAD

**Discussion by hcg1023** [react-hook-form/react-hook-form#10296] [Q&A]
I have a form, it is like this `{ 'a.b': '1' }`, when I specify the name as `a.b`, the result I get is `{ a: {b: '1' } }`

what should I do?

**Comment by hcg1023** [comment]
https://codesandbox.io/s/react-hook-form-testing-forked-vt5jjg?file=/src/App.tsx

**Comment by hcg1023** [comment]
<img width="1096" alt="image" src="https://user-images.githubusercontent.com/47942469/232771161-7a03bdcf-8910-42fd-8ebc-11a42f083173.png">

**Comment by bluebill1049** [comment]
dot notions are designed for the nested objects.

**Reply by hcg1023** [reply]
so i can't set the field name to `a.b` right？
